[2024-07-02 00:54:37,535: INFO: main: Welcome to cnn classifier]
[2024-07-03 01:37:36,137: INFO: main: >>>>>>>>>stage Data Ingestion Stage started <<<<<<<<<<<]
[2024-07-03 01:37:36,138: ERROR: main: [Errno 2] No such file or directory: '..\\config\\config.yaml']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    obj.main()
  File "d:\project\chest cancer classification\end-to-end-chest-cancer-classification-using-mlflow-and-dvc\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 13, in main
    config = ConfigurationManager()
  File "d:\project\chest cancer classification\end-to-end-chest-cancer-classification-using-mlflow-and-dvc\src\cnnClassifier\config\configuration.py", line 13, in __init__
    self.config = read_yaml(config_filepath)
  File "C:\Users\keyur\miniconda3\envs\mlflowproj\lib\site-packages\ensure\main.py", line 849, in __call__
    return_val = self.f(*args, **kwargs)
  File "d:\project\chest cancer classification\end-to-end-chest-cancer-classification-using-mlflow-and-dvc\src\cnnClassifier\utils\common.py", line 37, in read_yaml
    raise e
  File "d:\project\chest cancer classification\end-to-end-chest-cancer-classification-using-mlflow-and-dvc\src\cnnClassifier\utils\common.py", line 30, in read_yaml
    with open(path_to_yaml) as yaml_file:
FileNotFoundError: [Errno 2] No such file or directory: '..\\config\\config.yaml'
[2024-07-03 01:38:37,382: INFO: main: >>>>>>>>>stage Data Ingestion Stage started <<<<<<<<<<<]
[2024-07-03 01:38:37,382: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-07-03 01:38:37,382: INFO: common: yaml file: params.yaml loaded successfully]
[2024-07-03 01:38:37,382: INFO: common: created directory at: artifacts]
[2024-07-03 01:38:37,382: INFO: common: created directory at: artifacts/data_ingestion]
[2024-07-03 01:38:37,397: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1iLLcNSi9yre-OdgUqP4_JBTf7V1XERKa/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-03 01:39:04,496: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1iLLcNSi9yre-OdgUqP4_JBTf7V1XERKa/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-07-03 01:39:06,427: INFO: main: >>>>>>>>>Data Ingestion Stage Completed <<<<<<<<<<<]
